src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,43 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    user,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
+        </p>
```

Comment:
```suggestion
        { user && (
          <p>
            {' Posted by  '}

            <UserInfo user={user} />
          </p>
        )}
```
technically user can be undefined. suggest nor render this block in such case